Formula/testthingmac.rb

```diff
@@ -0,0 +1,51 @@
+class Testthingmac < Formula
+  desc "DeskThing for Mac - Manual SHA256 Formula"
+  homepage "https://github.com/ItsRiprod/TestThing"
+  version "0.10.76"
+  license "Proprietary"
```

Comment:
Sorry, proprietary software is not allowed in the official core tap [^1].

[^1]: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae